projects/go-lib/src/lib/components/go-off-canvas/go-off-canvas.component.html

```diff
@@ -11,13 +11,8 @@
   role="dialog"
 >
   <div class="go-off-canvas__header">
-    <button
-      (click)="closeOffCanvas()"
-      class="go-off-canvas__button"
-      type="button"
-    >
-      Close
-    </button>
+    <h2>{{ header }}</h2>
+    <go-icon icon="clear" iconModifier="medium" iconClass="clickable" (click)="closeOffCanvas()"></go-icon>
```

Comment:
@deotte we should actually switch this to be a `go-icon-button`. You may run into troubles with icon color and what not, but there is an issue open for that if you'd be interested in picking that one up as well. #263